subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/CustomPluginIntegrationTest.groovy

```diff
@@ -209,38 +198,31 @@ class CustomPluginTest {
     @Test
     public void test() {
         def project = ProjectBuilder.builder().build()
-
```

Comment:
Same thing for this test too.  GradleImplDepsVisibilityIntegrationTest covers it